examples/03_datetime_encoder.py

```diff
@@ -148,14 +160,21 @@
 # For prediction tasks, we recommend using the |TableVectorizer| inside a
 # pipeline, combined with a model that can use the features extracted by the
 # |DatetimeEncoder|.
-# Here we'll use a |RidgeCV| model as our learner.
-#
-from sklearn.ensemble import HistGradientBoostingRegressor
+# Here we'll use a |RidgeCV| model as our learner. We also fill null values with
+# |SimpleImputer| and then rescale numeric features with |StandardScaler|.
+
+from sklearn.impute import SimpleImputer
+from sklearn.linear_model import RidgeCV
 from sklearn.pipeline import make_pipeline
+from sklearn.preprocessing import StandardScaler
 
-pipeline = make_pipeline(table_vec, HistGradientBoostingRegressor())
-pipeline_weekday = make_pipeline(table_vec_weekday, HistGradientBoostingRegressor())
-pipeline_periodic = make_pipeline(table_vec_periodic, HistGradientBoostingRegressor())
+pipeline = make_pipeline(table_vec, StandardScaler(), SimpleImputer(), RidgeCV())
+pipeline_weekday = make_pipeline(
+    table_vec_weekday, StandardScaler(), SimpleImputer(), RidgeCV()
+)
+pipeline_periodic = make_pipeline(
+    table_vec_periodic, StandardScaler(), SimpleImputer(), RidgeCV()
+)
```

Comment:
Please add a small text that says that we'll compare different encoding of datetime, for the linear model